src/App/Resources/zh-CN/Resources.resw

```diff
@@ -426,7 +426,7 @@
   <data name="MaybeDeleted" xml:space="preserve">
     <value>可能已被删除。</value>
   </data>
-  <data name="NetworkButton.[using:Windows.UI.Xaml.Automation]AutomationProperties.Name" xml:space="preserve">

```

Comment:
Don't edit the zh-tw & zh-cn resource files. They are autogenerated by the touchdown build.

https://www.osgwiki.com/wiki/Factory_Orchestrator_Developer_Information#Localization_Build